core/src/main/groovy/noe/common/utils/processid/ProcessUtils.groovy

```diff
@@ -67,13 +71,24 @@ public final class ProcessUtils {
                 f.setAccessible(true);
                 processId = f.getInt(process);
```

Comment:
Is this OK? I can see the implementation of windows-specific function was updated, but not this one.

Reply:
Hm, good point - I updated this to `getLong` too, since the field is `Long` in new JDKs implementations. Still this should not break with the JDK8 hopefully :)